.github/workflows/pr-check-shell.yaml

```diff
@@ -0,0 +1,15 @@
+name: Check Shell Code
```

Comment:
Keep the name in same format with other action yaml file for CNAP, also keep the file name in same format with other action yaml file.